exec/java-exec/src/main/java/org/apache/drill/exec/vector/BitVector.java

```diff
@@ -211,8 +220,11 @@ public TransferPair makeTransferPair(ValueVector to) {
 
   public void transferTo(BitVector target) {
     target.clear();
+    if (target.data != null) {
+      target.data.release();
```

Comment:
Originally I did change all the release()s and retain()s to (1) in all files. But Jacques made me undo them in a few files he reviewed in another patch, and I haven't searched to see if there are new additions since I made the original changes. I think it's worth saving the function call for some low-level things (such as value vectors). What would you prefer?